docs/source/resources/define-metadata.rst

```diff
@@ -1769,16 +1769,23 @@ to load data from other files. These are ``load_setup_py_data``, ``load_file_reg
       name: foo
       version: {{ version }}
 
-* ``load_file_regex``: Searches a file for a regular expression and returns the
-  first match as a Python ``re.Match object``. For example::
+* ``load_file_regex(load_file, regex_pattern, from_recipe_dir=False) -> re.Match | None``: Search a file for a regular expression returning the
+  first match as a Python `re.Match
+  <https://docs.python.org/3/library/re.html#match-objects>`_ object.
+
+  For example::
+
+    {% set version_match = load_file_regex(
+      load_file="conda_package_streaming/__init__.py",
+      regex_pattern='^__version__ = "(.+)"') %}
+    {% set version = version_match[1] %}
```

Comment:
```suggestion
* ``load_file_regex``: Search a file for a regular expression returning the
  first match as a Python `re.Match
  <https://docs.python.org/3/library/re.html#match-objects>`_ object.

  For example, using ``load_file_regex(load_file, regex_pattern, from_recipe_dir=False) -> re.Match | None``::

    {% set version_match = load_file_regex(
      load_file="conda_package_streaming/__init__.py",
      regex_pattern='^__version__ = "(.+)"') %}
    {% set version = version_match[1] %}
```
This is just one idea for keeping the consistency of introducing the basic function names, while still keeping this full example intact. Would love others' opinions.